.github/tests/vrt/template-paths.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This is the default scenarios collection that will be used if there is not a theme-specific scenarios file in place
+ * for a given theme. You can copy this file and then name it `theme-paths.js`.
+ *
+ */
+
+/**
+ * Stores the scenarios for each page/endpoint that should be tested
+ * @type {{}}
+ */
+var scenarioPaths = {};
+
+/**
+ * For each page/endpoint you want to test, create a new array entry that contains at least the keys/properties `label`
+ * and `path`.
+ *
+ * additional properties you can set for each scenario are documented here: https://github.com/garris/BackstopJS#advanced-scenarios
+ *
+ * However, do NOT set `referenceUrl` or `url` as those will be overridden
+ *
+ * `path` should assume the URL ends in a trailing slash. For example, if the page you want to test against is
+ * https://master-7rqtwti-fqfjrmtjbjta4.eu-3.platformsh.site/a/path/to/foo/bar/
+ * Then for `path` it should be "a/path/to/foo/bar/"
+ *
+ * @type {{path: string, label: string}[]}
+ */
+scenarioPaths.paths = [
+    {
+        "label":"MjY0MjUK",
+        "path": "2021/12/01/mjy0mjuk/"
```

Comment:
Do we a date ? 
or maybe i'm wrong, is it just an example ?

Reply:
Ah. Good question. 

This corresponds to the address to be tested: https://master-7rqtwti-fqfjrmtjbjta4.eu-3.platformsh.site/2021/12/01/mjy0mjuk/ 

The label is the page's title. These are a bit more confusing since all of the data in the test project is random.

in the template-paths.js file we only include the path to the endpoint since the script appends the domain (reference/testing) to each path when it starts the test.